### PR TITLE
Adding Women Safety Numbers State wise

### PR DESCRIPTION
**Pull request by naman153**
## Related Issue or bug
  - Info about Issue or bug

Fixes: #257 [issue number that will be closed through this PR]

#### Describe the changes you've made
A clear and concise description of what you have done to successfully close your assigned issue. Any new files? or anything you feel to let us know!

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
A clear and concise description of it.

## Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots

 Original           | Updated
 :--------------------: |:--------------------:
 **original screenshot**  | <b>updated screenshot

 </b> |

### CODE REVIEW

**Comment by naman153** [comment]
Have a look on this and let me know if any changes are to be made.

**Review by swapnilsparsh — CHANGES_REQUESTED** [review]
Remove the extra space below the state name.

**Comment by naman153** [comment]
Changes you had requested for. Are they okay now or some change had to be done.

Original           | Updated
 :--------------------: |:--------------------:
 ****  | <b>

</b>

**Review by swapnilsparsh — CHANGES_REQUESTED** [review]
Annotation 2021-04-19 233309

There is no difference between heading and details that are there so you might remove the bold styling from details or reduce the size of the details for differentiation.

**Comment by naman153** [comment]
> Annotation 2021-04-19 233309
> 
> There is no difference between heading and details that are there so you might remove the bold styling from details or reduce the size of the details for differentiation.

| Updated
 :--------------------: |:--------------------:




Changes you had requested for. Are they okay now or some change had to be done.

File: main_app/templates/main_app/helpline_numbers.html [hunk 1]
```diff
@@ -208,9 +212,1025 @@ <h1 class="wb-stl-heading1">
...
+    </div>
+    
+
+    <div class="container">
+            
+        <ul class="collapsible popout" data-collapsible="accordion">
+            <li>
+                <div class="collapsible-header">
+                    <i class="material-icons">arrow_drop_down</i>
+                    <p style="height: 30px; padding-top: 35px;" width="275" class="wb-stl-normal"><span class="wb-stl-special"><span> &nbsp; &nbsp; <h3><strong>DELHI</strong></h3></span></span></p>
```
Comment (sakshikhachane) [inline main_app/templates/main_app/helpline_numbers.html:245]: - [ ] Change the font style of all the headings. Make it similar to the font style of `Women Helpline - All India`.
- [ ] Add hovering effect to the block of heading.

File: main_app/templates/main_app/helpline_numbers.html [hunk 2]
```diff
@@ -208,9 +212,1025 @@ <h1 class="wb-stl-heading1">
...
+                </div>
+                <div class="collapsible-body">
+                    <table class="table table-striped " cellpadding="20" cellspacing="4" style="background-color:white;color:black;font-family:lobster,lobster,sans-serif;font-size:3vw; width:100%; " width="1250px">
+                        <tbody class="womenhp2">
+                            <tr>
+                                <td style="height: 100px; padding-top: 35px;" width="275px">
+                                    <p class="wb-stl-normal"><span class="wb-stl-special"><span>&#8226; &nbsp; &nbsp;Delhi Commison for women</span></span></p>
```
Comment (sakshikhachane) [inline main_app/templates/main_app/helpline_numbers.html:252]: - [ ] Change the font style of all lists. Make it similar to the font style of `Women Helpline - All India`.
- [ ] The font size must be similar to the `Women Helpline - All India`.

**Review by sakshikhachane — CHANGES_REQUESTED** [review]

File: main_app/templates/main_app/helpline_numbers.html (hunk 2 above)
Comment (naman153) [reply main_app/templates/main_app/helpline_numbers.html:252]: - [x] Change the font style of all the headings. Make it similar to the font style of  `Women Helpline - All India.`
- [x] Add hovering effect to the block of headings.
- [x] Change the font style of all lists. Make it similar to the font style of `Women Helpline - All India.`
- [ ] The font size must be similar to the `Women Helpline - All India.`

@sakshikhachane Actually the size of list is made small to had a difference in header and list.
This thing is suggested by @swapnilsparsh. If you wish i can revert back the changes. 

Please have a look on the same and let me know for further correction if any.

**Review by sakshikhachane — APPROVED** [review]

**Comment by naman153** [comment]
> Thank you @naman153 for your contribution. Looking forward for more such amazing contributions :)

Thanks @swapnilsparsh @sakshikhachane for all the support and help you had provided me. Will look forward to contribute to it further for sure.

**Pull request merged**